### PR TITLE
lava testnet endpoints by🔥STAVR🔥

### PR DESCRIPTION
**Pull request by obajay**

### CODE REVIEW

**Comment by dylanschultzie** [comment]
please use an appropriate title

**Comment by reversesigh** [comment]
Please return chain.json to the filename `chain.json`

File: testnets/lavatestnet/chain.json [hunk 1]
```diff
@@ -116,22 +116,44 @@
         "provider": "Polkachu"
       }
     ]
+  },
```
Comment (reversesigh) [inline testnets/lavatestnet/chain.json:119]: I think this line isn't needed.

**Comment by JeremyParish69** [comment]
Sorry to add more, but you're adding endpoints above what looks like the developers' endpoints. Please only added nodes and interfaces to the bottom of arrays.

**Pull request merged**